.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: CI
+
+on: # yamllint disable-line rule:truthy
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+concurrency:
+  group: "${{ github.head_ref || github.ref }}-${{ github.workflow }}"
+  cancel-in-progress: true
+
+jobs:
+  lint-test:
+    name: Lint and Test
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version: ["3.12"]
+    
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Poetry
```

Comment:
No need to install poetry anymore as we're now using pip.

Reply:
Oops, thanks.